R/metrics-quantile.R

```diff
@@ -92,7 +92,7 @@ interval_coverage_quantile <- function(observed, predicted, quantile, range = 50
   assert_number(range)
   necessary_quantiles <- c((100 - range) / 2, 100 - (100 - range) / 2) / 100
   if (!all(necessary_quantiles %in% quantile)) {
-    rlang::warn(
+    warning(
```

Comment:
why are you changing this back?

Reply:
It errored 🥲 So I made an issue (#415) and switched it back for now. In hindsight I see your point about adding that as context here... Apologies!